webtau-cli/src/main/java/com/twosigma/webtau/cli/Cli.java

```diff
@@ -41,7 +41,7 @@ public ProcessEnv env(Map<String, String> env) {
     }
 
     public ProcessEnv env(String... keyValue) {
-        return new ProcessEnv(CollectionUtils.aMapOf(keyValue));
+        return new ProcessEnv(CollectionUtils.aMapOf((Object[]) keyValue));
```

Comment:
Why this cast?

Reply:
IDEA gave me a warning in that file that it is confusing. `unclear if a vararg or non-varargs call is desired`

Reply:
weird but ok.